ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/attestation/utils/AttestationBitsElectraTest.java

```diff
@@ -224,21 +231,70 @@ void aggregateOnMultipleOverlappingCommitteeBitsButWithSomeOfAggregationOverlapp
         .containsExactly(0, 1, 2, 3, 8);
   }
 
+  @Test
+  void aggregateSingleAttestationFillUp_notAggregatable() {
+    /*
+     01|234 <- committee 0 and 1 indices
+     10|100 <- bits
```

Comment:
Why splitting the aggregation bits with the `|` here?
The comments are a bit confusing to me:
`01|234 <- committee 0 and 1 indices`
01 are the committees bits
and
234 are the committees size ordered
right?

Reply:
> 0 has size 2 and 3 size 3

I guess you mean `0 has size 2 and 1 has size 3`